tests/ut/unique.d

```diff
@@ -112,6 +112,17 @@ mixin TestUtils;
     Class.numClasses.shouldEqual(0);
 }
 
+@("construct base interface")
+@system unittest {
+    auto allocator = TestAllocator();
+    {
+        Unique!(Interface, TestAllocator*) bar = Unique!(Class, TestAllocator*)(&allocator, 5);
```

Comment:
Where is `Interface` defined?

Reply:
Fixed (tests/ut/package.d)

This missed file was the real reason for tests failing. Currently large amount of *.lst files generated in the root directory and I missed that.

Reply:
I'm still not sure what this would do. I assume the references are all null and completely unusable? Before it didn't compile, but now this seems like it'd be a runtime segfault, no?

Reply:
No, it works. As I can see [here](https://github.com/atilaneves/automem/blob/a806e1febe1ce339e98302d2d0bb19e65f319a02/source/automem/unique.d#L185) this is implicit casting of Class to Interface. And this means memory leak is possible [here](https://github.com/atilaneves/automem/blob/a806e1febe1ce339e98302d2d0bb19e65f319a02/source/automem/unique.d#L175) because interfaces are just pointers.